doc/api/process.md

```diff
@@ -1819,6 +1819,41 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourceUsage()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object}
+    * `minflt` {integer}
+    * `majflt` {integer}
+    * `inblock` {integer}
+    * `oublock` {integer}
+    * `nvcsw` {integer}
+    * `nivcsw` {integer}
+
+The `process.resourceUsage()` method returns the resource usage
+for the current process.
+To understand what these value mean, please refer to
+[getrusage(2)](http://man7.org/linux/man-pages/man2/getrusage.2.html)
+On Windows platforms, only `inblock` and `oublock` are supported,
```

Comment:
Just noting that this sentence will need to be updated pending two decisions in this PR:

- Abbreviated or expanded field names.
- Inclusion of all the fields present in libuv.